google/cloud/bigquery/schema.py

```diff
@@ -556,3 +560,88 @@ def to_api_repr(self) -> dict:
         """
         answer = {"names": list(self.names)}
         return answer
+
+
+class SerDeInfo:
+    """Serializer and deserializer information.
+    Args:
```

Comment:
Might need a blank line here for the docs renderer to work?

```suggestion
    """Serializer and deserializer information.

    Args:
```

Reply:
Resolved.